### ISSUE TITLE
Possible accessibility issues with tabs component

### ISSUE THREAD

**Issue by timpaul** [alphagov/govuk-frontend#943]
We've had the following feedback from a user in HMRC:

> The Tabs component in the design system doesn’t fully work with IE 11 and JAWS. JAWS does not announce the number within the set (for example, tab 3 of 4).

> The Tab component does not work in iOS at all. VoiceOver sees the tabs as links instead, and there is also an H2 (Contents,) appearing before them. It seem to work fine with Chrome (Windows) and also on Safari (Mac).

**Comment by nickcolley** [comment]
As an update, the issues raised around Safari are due to the fact that on smaller screens the tabs are intentionally not initialised. This may need better communication in the Design System guidance.

**Comment by igloosi** [comment]
I couldn't get it to announce the number of tabs either on our example or this [one](http://a11yideas.com/testcode/tabs/tabs_with_ARIA.html) in IE11

Apparently, " It also (after the short-cut key) announces the “Tab 1 of 3” to indicate size and location." but i couldn't get that shortcut to work (it should have been capslock on laptop according to the sheet)

**Comment by nickcolley** [comment]
I also could not get the JAWs behaviour to trigger when using https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html

**Comment by nickcolley** [comment]
I have concluded that this is extra functionality that JAWs does not current have and have raised an issue https://github.com/FreedomScientific/VFO-standards-support/issues/117 requesting this feature as it's something useful seen in NVDA and VoiceOver.

If we were to add this context manually it would be read out twice for users that already contain this context.

Due to this, there's nothing additional we could be doing, so I'm going to close this out and hope the JAWs team consider adding this in a future release.

I'll let the original contributor of this issue know.
